styles/Canonical/025b-latinisms-to-reconsider.yml

```diff
@@ -0,0 +1,10 @@
+# 025 - Latinisms that we strongly advise against
+
+extends: existence
+message: Instead of 'a.m.', 'AM', 'p.m.', and 'PM', use 24-hour time.
+link: https://docs.ubuntu.com/styleguide/en#latin-words-and-phrases
+level: suggestion
+nonword: true
+tokens:
+  - '\b(?:a\.m\.(?!\w)|a.m(?![\.\w])|AM\b|A\.M\.(?!\w)|A\.M(?![\.\w]))'
```

Comment:
Good point, however when I did a brief search online it reminded me that forms like `6:30pm` also unfortunately exist, so it might be easier to prefix the whole block with `[0-9]?`.

Reply:
`a.m(?![\.\w])` and similar negative lookaheads are there to catch typos like `7 a.m in the morning` , but yes, the second char is supposed to be a literal, oops.

Reply:
True. You could change `A\.M\.(?!\w)` to `A\.M\.?(?!\w)` to extend this case to catch your example, instead of duplicating.

Reply:
This should be even more comprehensive: 451ff5a7fb495fbf320f3a91a999ecaddf84ed62